docs/how_tos/localized_currency.rst

```diff
@@ -0,0 +1,66 @@
+########################
+Localized Currency HOWTO
+########################
+
+************
+Introduction
+************
+
+This HOWTO explains what the localized currency cookie is, how it gets set, and how it is connected to our LocalizedPrice component.
+
+.. contents:: Table of Contents
+
+***************************
+What is localized currency?
+***************************
+
+This is when we show users prices on the checkout page in what we think is the correct currency for where they are. These prices can be rounded, since the final checkout charge will still be in U.S. dollars (USD).
+
+***********************
+When is the cookie set?
+***********************
+
+The ``edx-price-l10n`` cookie is set when a user visits a course's About page. (Since August 2019, the cookie is set whether the About page is generated by Drupal or Prospectus.)  This cookie is set based on the user's location as per the browser's `geolocation API <https://developer.mozilla.org/en-US/docs/Web/API/Geolocation_API>`_.
```

Comment:
Happened to see this, if it matters the cookie is now also set on all program about pages as well. Also we switched to using IP location from cloudflare.

Reply:
Good info, thanks!